tests/values-docker-compose.yaml

```diff
@@ -4,8 +4,10 @@ nodes:
     address: 'http://owkin.substra-backend:8000'
     user: 'admin'
     password: 'admin'
+    shared_path: '/tmp/substra/'
```

Comment:
The new test doesn't pass when using docker-compose, I get the following error:
```
E               substra.sdk.exceptions.InvalidRequest: 400 Client Error: Bad Request for url: http://owkin.substra-backend:8000/data_sample/: One of your paths does not exist, is not a directory or is not an absolute path: /tmp/substra//gngnvl7v/data-0
```

I guess this is because of the trailing slash on this line.

Reply:
Yes it has not been tested with docker-compose, I'm not sure where medias are located with the latest docker-compose and the latest mac update.

Reply:
You could use the `SUBSTRA_PATH` env variable for that with a default on `/substra`. This is what's used by the `start.py` script in the backend, so it should match.

Reply:
I think the tests setup should stay independent from the backend setup, so we should update directly the `values-docker-compose.yaml` file.

If really needed we could use a test env var for that but it should be named accordingly (`SUBSTRAT_SHARED_PATH`) .

Reply:
If we want to keep things simple, we could just remove the trailing slash so that the generated paths are `/tmp/substra/gngnvl7v/data-0` instead of `/tmp/substra//gngnvl7v/data-0`.

Reply:
What about this (until we have a better solution)

```diff
-    shared_path: '/tmp/substra/'
+   shared_path: '/tmp/substra' # needs to match SUBSTRA_PATH
```

Reply:
Looks like a good solution, will do that.

Reply:
👍